modules/common/src/test/scala/surge/internal/health/windows/actor/HealthSignalWindowActorSpec.scala

```diff
@@ -32,11 +32,11 @@ class HealthSignalWindowActorSpec
   val bus: HealthSignalBusInternal = HealthSignalBus(testHealthSignalStreamProvider(Seq.empty))
 
   override def afterAll(): Unit = {
-    TestKit.shutdownActorSystem(system)
+    TestKit.shutdownActorSystem(system, verifySystemShutdown = true)
   }
 
   "HealthSignalWindowActor" should {
-    "tick" in {
+    "tick" ignore {
```

Comment:
flaky test unfortunately

Reply:
Should be fixed on less flaky with #47 but I'm fine ignoring if this one makes it in first